artifactory/1.11/artifactory-enterprise.md

```diff
@@ -0,0 +1,142 @@
+# Setting up Artifactory Enterprise on DC/OS
+
+[Artifactory Enterprise](https://www.jfrog.com/artifactory/versions/#High-Availability) is a highly available installation of Artifactory. It does this by using a load balancer to balance requests across multiple Artifactory instances.
+
+![Artifactory Enterprise Architecture](img/HA_Diagram.png)
+
+## Prerequisites
+
+- DC/OS 1.11 or later with at least one public agent
+- [DC/OS CLI installed](https://docs.mesosphere.com/1.11/cli/install/) and
+  configured to use your cluster
+- Database ([Postgres](install-postgres.md), [MySQL](install-mysql.md), Oracle, or MS SQL Server)
+- Artifactory Enterprise license
+- NFS directory mounted to each node
+
+## Setting up Artifactory Enterprise
+
+### Set up storage on each node (required for high availability)
+
+All nodes need to be able to read and write to the same file share. We recommend
+using NFS (or an equivalent like [Amazon's EFS](https://aws.amazon.com/efs/)).
+
+For example, using the default location of `/var/artifactory`, mount an example
+file share on each DC/OS private agent using the following command:
+
+```
+sudo mount artifactoryha.mount.com:/artifactory /var/artifactory/
+```
+
+You must ensure this mount is writeable.
+
+### Setting up Artifactory Enterprise
+
+1. Create a new file called `artifactory-enterprise-options.json` with the
+   following content.
+
+Be sure to:
+
+- replace `service.licenses` with your own license string. Note: Licenses can be configured in the Artifactory UI.
+- replace `service.database.user` and `service.database.password` with the
+  correct credentials if you have customised these values.
+- replace `artifactory` within `service.database.url` with the correct database name
+  if you have used a different one.
+- replace `enterprise.master_key` with your own master key value.
+
+```
+{
+  "service": {
+    "name": "artifactory",
+    "licenses": "replaceme",
+    "host-volume": "/var/artifactory",
+    "database": {
+      "type": "postgresql",
+      "host": "postgresql.marathon.mesos",
+      "port": 5432,
+      "url": "jdbc:postgresql://postgresql.marathon.mesos:5432/artifactory",
+      "user": "jfrogdcos",
+      "password": "jfrogdcos"
+    }
+  },
+  "enterprise": {
+    "enabled": true,
+    "master_key": "replaceme"
```

Comment:
What is the master_key? I only have license keys, and previous examples (for 1.10, etc) didn't include this field.

Reply:
'master_key' is use used in setting up the Artifactory HA cluster.
It is AES 128 bit secret key. Generate master.key using command '$openssl rand -hex 16'
Will clarify by adding more documentation on it.